project/Build.scala

```diff
@@ -221,13 +232,6 @@ object DottyBuild extends Build {
     ).
     settings(
       addCommandAlias(
-        "packageAll",
-        ";dotty-interfaces/package" +
-        ";dotty-compiler/package" +
-        ";dotty-library/package" +
-        ";dotty-compiler/test:package"
-      ) ++
-      addCommandAlias(
         "partest",
         ";packageAll" +
```

Comment:
Don't use `Def.sequential` unless you know exactly what you are doing.

`Def.sequential` does not give much *guarantee* in terms of sequentiality. In particular, if from X you depend on task A which depends on a sequence of B, C; but at the same time X depends on C (via another path of dependencies); then B and C will probably still run in parallel.